tower-buffer/src/lib.rs

```diff
@@ -43,6 +43,8 @@ pub struct ResponseFuture<T, E> {
     state: ResponseState<T, E>,
 }
 
+type Error = Box<StdError + Send + Sync>;
```

Comment:
This is a bit of a nit, but I'd prefer we just refer to this type explicitly, and avoid the renaming import of `std::error::Error`...

Reply:
I personally prefer this (though I may have just referenced `std` directly instead of having a `use` statement, but that matters less to me. Repeatedly referencing `Box<Error + Send + Sync>` feels like code dup to me.

Reply:
I did `Box<Error + Send + Sync>` first, and then I switched to this type alias. I think the type alias definition being:

```rust
type Error = Box<std::error::Error + Send + Sync>
```
...would probably be best. Optionally, it could be placed in the root Tower crate, but I don't feel too strongly about it.

Reply:
It's the renaming import that I'm really not fond of, but it's not particularly important to me...